### PR TITLE
[Enhancement] Improve get_started documents and bump version to 0.7.0

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Refactor get_started documents according to internal reviews

## Modification

en/get_started.md
zh_cn/get_started.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started.md [hunk 1]
```diff
@@ -243,11 +315,6 @@ cv2.imwrite('output_detection.png', img)
 
 You can find more examples from [here](https://github.com/open-mmlab/mmdeploy/tree/master/demo/python).
 
-```{note}
-If you build MMDeploy from source, please add ${MMDEPLOY_DIR}/build/lib to the environment variable PYTHONPATH.
-Otherwise, you will run into an error like ’ModuleNotFoundError: No module named 'mmdeploy_python'
-```
-
 #### C API
```
Comment (lzhangzz) [inline docs/en/get_started.md:318]: Maybe we can replace the C API example with a C++ one, which is simpler and less verbose.

File: docs/en/get_started.md [hunk 2]
```diff
@@ -67,125 +63,194 @@ We recommend that users follow our best practices installing MMDeploy.
...
 ```
 
-In the next chapters, we are going to present our 'Hello, world' example based on the above settings.
+</details>
+
+<details>
+<summary><b>Windows, CUDA 11.x, TensorRT 8.2.3.0</b></summary>
 
-For the installation of all inference backends supported by MMDeploy right now, please refer to:
+请在 Conda Prompt Shell 中执行以下命令
```
Comment (tpoisonooo) [inline docs/en/get_started.md:188]: 这是个 en 文档。

File: docs/en/get_started.md [hunk 3]
```diff
@@ -318,16 +383,22 @@ When you build this example, try to add MMDeploy package in your CMake project a
...
 find_package(MMDeploy REQUIRED)
-mmdeploy_load_static(${YOUR_AWESOME_TARGET} MMDeployStaticModules)
-mmdeploy_load_dynamic(${YOUR_AWESOME_TARGET} MMDeployDynamicModules)
-target_link_libraries(${YOUR_AWESOME_TARGET} PRIVATE MMDeployLibs)
+target_link_libraries(${name} PRIVATE mmdeploy ${OpenCV_LIBS})
 ```
 
 For more SDK C API usages, please read these [samples](https://github.com/open-mmlab/mmdeploy/tree/master/demo/csrc).
 
+#### C++ API
```
Comment (tpoisonooo) [inline docs/en/get_started.md:391]: C++/C#/Java/..  给个小的表格就行，不用写 3 句。

**Review by tpoisonooo — APPROVED** [review]
别的 LGTM。

**Review by AntiLeaf — APPROVED** [review]

**Review by irexyc — APPROVED** [review]

**Review by lzhangzz — APPROVED** [review]
LGTM

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

File: docs/en/01-how-to-build/android.md [hunk 4]
```diff
@@ -174,7 +173,7 @@ make -j$(nproc) install
 </tbody>
 </table>
 
-#### Build SDK
+#### Build SDK and Demos
```
Comment (RunningLeon) [inline docs/en/01-how-to-build/android.md:176]: maybe no plural and  `Demo` is OK.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**